apps/server/src/controllers/dictionaryController.ts

```diff
@@ -105,7 +107,7 @@ export const getSchema = async (req: Request<{ dictId: string; schemaName: strin
 	const dictionary = await dictionaryService.getOneById(dictId);
 	const formattedDictionary = replaceReferences(dictionary);
 
-	const schema = formattedDictionary.schemas.find((schema) => schema.name === schemaName);
+	const schema = formattedDictionary.schemas.find((schema: { name: string }) => schema.name === schemaName);
```

Comment:
Why is this type declaration necessary? Is it not inferring the type of schema from `formattedDictionary.schemas`?